### PR TITLE
Documentation Update Part 3

### PR DESCRIPTION
**Pull request by paulhazen**
### TL;DR

This PR finalizes the documentation update by splitting the `README.md` into separate sections and replacing them with links, although note that GitHub is currently addressing an issue with broken relative links in banner sections. Please share your thoughts on these changes for a constructive dialogue.

 
### Overview

While documentation is always a living ever-evolving creature - this PR represents the last in a series of updates to our documentation that aims to improve readability and the ease with which our users can find answers to their most frequent questions, and most quickly get started with the plugin. This PR _exclusively_ limits it's changes to splitting the README document into a variety of new documents and changing the README.md file to serve as a landing page / navigation page so that users can more quickly find the information they are looking for.

### Feedback Requested

Pay close attention to what is now _missing_ from the README file, and be certain that if content has been removed, it is appropriately highlighted in the form of a link to a different document. A lot of this may be subjective - so please _be_ subjective about what you think is correct or incorrect, so we can have a healthy discussion around it.

### Relative links within banner sections are weird

There is one tiny detail of this PR that is a little bit wonky. Currently, GitHub's implementation of the banner concept (text that's within a "NOTE," "WARNING", or "IMPORTANT" section colored blue, yellow, or purple respectively), any _relative_ links within do not properly resolve relative to the repository, but relative to GitHub's TLD. This is a known issue that GitHub is currently working on addressing. The solution I leaned into was to make the links absolute _to the branch being merged into_. There were about 3-4 links to which this happens, which we should keep an eye out for. If someone is viewing the documentation from the `development` branch, the links will work correctly. When we move `development` into `stable`, we will need to fix those links. Hopefully GitHub will resolve this issue sooner than later, since maintaining this is not sustainable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,41 +15,18 @@ See [this](/docs/plugin_advantages.md) for a more complete overview of the advan
...
-- Nintendo Switch (for details, please see the Nintendo Developer Portal)
-
-We are always looking to add the functionality of the plugin to more platforms, and have functionality for some in private development
+# Getting Started
 
-[^1]: Social overlay feature is currently not supported on this platform.
+## Prerequisites
 
-## Unsupported:
+* [Create an Epic Games Account](https://www.epicgames.com/id/register) (_although, most [features](#exploring-supported-eos-features) do not require an Epic Games Account_).
```
Comment (andrew-hirata-playeveryware) [inline README.md]: I think it might be import that it's required to have an epic account for integrating the plugin into a project and for testing it.

File: README.md [hunk 2]
```diff
@@ -90,202 +67,29 @@ There are many EOS features that do not require your player to have an Epic Game
...
-* Via a [tarball](#adding-the-package-from-a-tarball) _(easiest to get started quickly)_.
-* From a [git url](#adding-the-package-from-a-git-url) _(this method has the possible advantage of keeping the plugin up-to-date, if that's something that you would prefer)_.
-
-### Adding the package from a tarball
-
-1. Download the latest release tarball, `"com.playeveryware.eos-[version].tgz"` [here](https://github.com/PlayEveryWare/eos_plugin_for_unity/releases).
-
-    > [!WARNING]
-    > Do *not* attempt to create a tarball yourself from the source, unless you know what you are doing with respect to [Git LFS](https://docs.github.com/en/repositories/working-with-files/managing-large-files/configuring-git-large-file-storage).
+* Via a [tarball](/docs/add_plugin.md#adding-the-package-from-a-tarball) _(easiest to get started quickly)_.
```
Comment (andrew-hirata-playeveryware) [inline README.md]: I still believe we should specify that it is a UPM package not just a tarball. Github has attached to each release a source tarball which I would like to ensure that people don't confuse or conflate.

**Review by andrew-hirata-playeveryware — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (paulhazen) [reply README.md]: Good point. I'll make that more clear.

File: README.md (hunk 2 above)
Comment (paulhazen) [reply README.md]: As much as I agree with the potential for confusion, the UI within Unity Package Menu specifically refers to it as a tarball. I believe we are pretty clear about how to do this in greater detail on the [add_plugin.md](https://github.com/PlayEveryWare/eos_plugin_for_unity/blob/docs/doc-update/docs/add_plugin.md) document, can you check to make sure those instructions line up with your understanding? They should not have changed but I want to be extra certain.

Comment (andrew-hirata-playeveryware) [reply README.md]: Maybe we could call it a UPM tarball?

**Review by andrew-hirata-playeveryware — DISMISSED** [review]

File: docs/eos_features.md
Comment (Raxnam) [inline docs/eos_features.md:1]: May be worth using the check and Xs on the required EGS row as well instead of yes and no

**Review by Raxnam — CHANGES_REQUESTED** [review]

File: docs/eos_features.md
Comment (paulhazen) [reply docs/eos_features.md:1]: I considered that - but from a visual perspective I wanted to highlight the fact that whether the feature is supported or not is what's important. I initially did exactly as you are suggesting, but I felt like it made it confusing at a glance to know for sure which features were or were not supported. If there was a more subtle image, like a neutral (uncolored) check mark, I could replace the "Yes" values in that column with that check and replace the "No" values with empty space. Thoughts?

Comment (paulhazen) [reply docs/eos_features.md:1]: I think it looks a lot better now, good suggestion to make it more visually engaging.

Comment (Raxnam) [reply docs/eos_features.md:1]: yea I think that works quite well

**Review by Raxnam — APPROVED** [review]

**Review by andrew-hirata-playeveryware — APPROVED** [review]
